src/plugins/files/server/file_client/file_metadata_client/adapters/es_index.ts

```diff
@@ -82,11 +81,36 @@ export class EsIndexFilesMetadataClient<M = unknown> implements FileMetadataClie
   }
 
   async get({ id }: GetArg): Promise<FileDescriptor<M>> {
-    const { _source: doc } =
-      (await fetchDoc<FileDocument<M>>(this.esClient, this.index, id, this.indexIsAlias)) ?? {};
+    const { esClient, index, indexIsAlias } = this;
+    let doc: FileDocument<M> | undefined;
+
+    if (indexIsAlias) {
```

Comment:
Removed `fetchDoc()` utility since it was only being used here and moved the implementation here.